assets/css/NewEvaluation.less

```diff
@@ -0,0 +1,148 @@
+body {
+box-sizing: border-box;
+background-color: #f7f7f7;
+
+}
+
+.borderSwitch{
+border-bottom: solid thick #03a9f4;
+text-align: center;
+}
+
+.pgSwitch{
+text-decoration: none;
+color:black;
+font-size: 170%;
+margin-left: 5rem;
+padding-left: 2rem;
+padding-right: 2rem;
+}
+
+.homeLogo {
+margin-right: 9rem;
+margin-left: 2rem;
+}
+
+.header-img {
+width: 200px;
+}
+
+.dataForm {
+width: 100%;
+text-align: left;
+padding: 15px;
+margin-bottom: 25px;
+
+input{
+    text-align: left;
+    width: 32%;
+    min-height: 25px;
+    box-shadow: 0px 1px -15px -10px black inset;
+    border-bottom: 1px solid #ccc;         
+}
+}
+
+.table{
+width: 100%;
+height: 5rem;
+border-collapse: collapse;
+text-align: center;
+
+.row1{
+
+    background-color: #03a9f4;
+    color: white;
+}
+
+.row2{
+    border: solid thin black;
+
+    input{
+        background-color: white;
+    }
+}
+}
+
+.textA {
+resize: none;
+width:100%;
+background-color: #f7f7f7;
+}
+
+.mainForm {
+margin: 0 20% 0 20%;
+width: 66.66%;
+display: block;
+}
+
+fieldset {
+border: solid thin black;
+margin: 25px 0 25px 0;
+font-size: 120%;
+
+legend{
+    font-weight: bold;
+}
+}
```

Comment:
Does this closing curly brace have a corresponding opening brace? Hard to tell without proper code formatting and without looking at the whole file.